pkg/process/util/config_linux.go

```diff
@@ -0,0 +1,16 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+package util
+
+import "github.com/DataDog/datadog-agent/pkg/config/model"
+
+// GetRunInCoreAgentConfig returns the config value for process_config.run_in_core_agent.enabled.
+// Will always return false outside of linux.
```

Comment:
This comment may easily become outdated, I think we can probably remove it

Reply:
My linter was yelling at me 😢

Reply:
I meant just the `Will always return false outside of linux`

Reply:
removed

Reply:
Ah ok

Reply:
bringing it back